packages/utils/src/get-section-from-schedules.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async ({ content, siteId, load }) => {
+  const { sectionQuery } = content;
```

Comment:
Add `if (!sectionQuery.length) return null;` as the first line, then delete the `if` wrapper on L8 and the other `return null` on L12